pkg/tfgen/docs_test.go

```diff
@@ -1010,53 +965,21 @@ This is a test for CUSTOM_REPLACES.`)
 				},
 			}
 
-			c.providerInfo.DocRules = &tfbridge.DocRuleInfo{
+			tc.providerInfo.DocRules = &tfbridge.DocRuleInfo{
 				EditRules: func(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
 					return append([]tfbridge.DocsEdit{rule}, defaults...)
 				},
 			}
 		}),
-
```

Comment:
This tc was redundant?

Reply:
This test case was kept on L954:
```
		test("azurerm-sql-firewall-rule"),
```

I have moved the expected block to `test_data/azurerm-sql-firewall-rule/expected.json`.